SF iOS/SF iOS/Location/TravelTime/TravelTime+Arrival.m

```diff
@@ -13,6 +13,10 @@ @implementation TravelTime (Arrival)
 
 - (Arrival)arrivalToEventWithStartDate:(NSDate *)startDate endDate:(NSDate *)endDate {
     // if the event is in the past, magnitudes do not matter
+    if (self.travelTime < 0) {
```

Comment:
isn't this equivalent to `ArrivalAfterEvent`?

Reply:
This is perhaps true from a practical standpoint in that you're not going to be able to make the scheduled event either way. But it can be helpful to distinguish between "You're going to arrive a few minutes after the end of the event" (still might be some people there) and ["You are not going to space today."](https://xkcd.com/1133/) Whether there's any appreciable utility in providing directions to an event that has passed is probably a discussion to be had another time...

Incidentally, not sure how this ended up _after_ the comment. Going to fix that.